### PR TITLE
Misc queue from 2018 (sorry, lost sources) 

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -8282,7 +8287,9 @@ irreversable->irreversible
 is'nt->isn't
 isconnection->isconnected
 iserting->inserting
+isle->aisle
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: isle = small island

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -8304,6 +8311,7 @@ istead->instead
 istener->listeners
 ists->its, lists,
 isue->issue
+it's own->its own
```
Comment (arm-in) [inline codespell_lib/data/dictionary.txt]: whitespace

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: removed

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -9903,6 +9903,7 @@ intialze->initialize
 intialzed->initialized
 intialzing->initializing
 intimite->intimate
+intitial->initial
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the plural and initialled.

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -9911,6 +9912,7 @@ intot->into
 intpreter->interpreter
 intputs->inputs
 intrduced->introduced
+intreeg->intrigue
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add intrigued and intriguing too.

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -9939,6 +9941,8 @@ intruments->instruments
 intrusted->entrusted
 intstead->instead
 intuative->intuitive
+inturpratasion->interpretation
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add inturprat to interpret.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -9939,6 +9941,8 @@ intruments->instruments
...
+inturprett->interpret
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Add the tion version of this misspelling.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -9974,6 +9978,7 @@ invlisible->invisible
 invlove->involve
 invloved->involved
 invloves->involves
+invokable->invocable
 invokation->invocation
 invokations->invocations
 invokee->invoked
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also suggest invoke, although I'd wonder if invokee could be valid as the thing being invoked, but dictionaries don't seem to agree with it.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -10039,6 +10045,7 @@ iteratered->iterated
 iteratons->iterations
 iterface->interface
 iterfaces->interfaces
+iterm->term
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Also intern maybe?

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: What?

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 6 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: inturpratation->interpretation

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry I don't think this should suggest inturprat, especially given it's a typo!

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, this wasn't aimed at you edit, but something I saw a few lines below (hence where the comment was), can you drop it from invokable->invocable and add invokee->invoked, invoke,

**Comment by Gelma** [comment]
Done! I'm going to upload new version

File: codespell_lib/data/dictionary.txt (hunk 7 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

File: codespell_lib/data/dictionary.txt (hunk 8 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added

File: codespell_lib/data/dictionary.txt (hunk 4 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Sorry, I mean intreeged->intrigued etc.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

Comment (peternewman) [reply codespell_lib/data/dictionary.txt]: Can we also have intreeged->intrigued and intreeging->intriguing, I suspect dropping the u is a likely equivalent typo based on the genuine one you've found.

Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: done

**Comment by Gelma** [comment]
> LGTM, can you fix the conflict please @Gelma .

Done

**Comment by peternewman** [comment]
> > LGTM, can you fix the conflict please @Gelma .
> 
> Done

The dictionary needs sorting now @Gelma

**Comment by Gelma** [comment]
> > > LGTM, can you fix the conflict please @Gelma .
> > 
> > 
> > Done
> 
> The dictionary needs sorting now @Gelma

Done. Thanks

**Comment by Gelma** [comment]
I'm trying to keep updated and ready rebased all the entries in the first page of PRs.

**Comment by peternewman** [comment]
Appveyor timed out, but merging anyway. @larsoner is Appveyor not a required status?

**Comment by larsoner** [comment]
No it's not, mostly because of stuff like that (it's less reliable than Travis)

**Pull request merged**